media/css/externalpages/pocket/components/_footer.scss

```diff
@@ -4,11 +4,14 @@
 
 @import '../utils/variables';
 
-$image-path: '/media/img/externalpages/pocket';
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+$image-path: '/media/protocol/img';
+$pocket-image-path: '/media/img/externalpages/pocket';
```

Comment:
I had to use two different image paths, so I named the one holding pocket paths $pocket-image-path. However, in other files, we do have $image-path holding the pocket paths.